lang/python/python-dateutil/Makefile

```diff
@@ -15,33 +15,51 @@ PKG_LICENSE:=BSD-2-Clause
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/python-dateutil
 PKG_HASH:=88f9287c0174266bb0d8cedd395cfba9c58e87e5ad86b2ce58859bc11be3cf02
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-dateutil-$(PKG_VERSION)
 
+include $(INCLUDE_DIR)/host-build.mk
```

Comment:
this doesn't call any `$(eval $(call HostBuild))`; so there may not be any reason to have this include `host-build.mk`

Reply:
Oops.  Yeah, that one was a slip.  (Was borrowing from another file while debugging).